packages/blockTools/src/RenderHtml.js

```diff
@@ -44,7 +44,7 @@ class RenderHtml extends React.Component {
             this.div = el;
           }
         }}
-        className={methods.makeCssClass(style)}
+        className={methods.makeCssClass([{ display: 'inline-block' }, style])}
```

Comment:
Is this breaking on existing Html blocks?

Reply:
No, if it is default `block` it is. I tested it on some larger projects and renders fine. Basically, an extra `div` was added where it was normally a string only, now we need to maintain the same flow as the string in order not to be breaking.